scripts/microbe_stage/organelle_components/movement_organelle.as

```diff
@@ -189,10 +189,10 @@ class MovementOrganelle : OrganelleComponent{
         auto renderNode = organelle.world.GetComponent_RenderNode(organelle.organelleEntity);
         if (renderNode !is null && IsInGraphicalMode())
         {
-            renderNode.Node.setPosition(membraneCoords);
+            renderNode.Node.setPosition(organellePos);
```

Comment:
Doesn't this make the flagella not jump to the edge? The old version worked with flagella placed within the middle of the cell and looking fine.

Reply:
this was me simply not changing something back after experimenting, tis fixed